docs/proposal/20200506-single-controller-multitenancy.md

```diff
@@ -538,6 +537,46 @@ spec:
     name: test-account-creds
 ```
 
+<strong><em>Permission CRD</em></strong>
+
+<em>ControllersPrincipalUsagePermission</em>
+
+When `principalRef` is not provided in AWSCluster, controller falls back to use controller credentials, 
+but those credentials tends to have more comprehensive permissions than workload user would need.
```

Comment:
`but those credentials tends to have more comprehensive permissions than workload user would need.`

I don't think that's strictly true - the user still doesn't have direct access to the credentials, I think the more accurate statement is along the lines of:

the controller's credentials are necessarily tied to an AWS account, and in many cases this is where the management cluster is deployed, and the cloud administrators will not want users to provision workload clusters in the management AWS account.